src/provider/remoteFileSystemProvider.ts

```diff
@@ -359,29 +359,44 @@ class VirtualFileSystem {
         if (fileType && fileType==='doc' && fileEntity) {
             const doc = fileEntity as DocumentEntity;
             const _content = JSON.stringify(new TextDecoder().decode(content));
+            let currentPos = 0;
```

Comment:
The `currentPos` and `hisCount` variables should be defined in the closure after `op: `.